tests/test_urlpatterns.py

```diff
@@ -98,7 +97,7 @@ def test_format_suffix_django2(self):
     def test_format_suffix_django2_args(self):
         urlpatterns = [
             path('convtest/<int:pk>', dummy_view),
-            re_path(r'^retest/(?P<pk>[0-9]+)$', dummy_view),
+            path('retest/<int:pk>', dummy_view),
```

Comment:
We shouldn't change this one. It's intentionally `re_path` in this one particular case, and the change to `int:pk` has failed one of the tests... https://travis-ci.org/github/encode/django-rest-framework/jobs/686013837

Reply:
i think in this case both path and re_path are used for test

Reply:
Yup.

Reply:
yeah completing this today :D sorry for the delay